cmd/gitserver/server/server.go

```diff
@@ -1515,18 +1517,20 @@ func (s *Server) exec(w http.ResponseWriter, r *http.Request, req *protocol.Exec
 
 		var tr *trace.Trace
 		tr, ctx = trace.New(ctx, "exec."+cmd, string(req.Repo))
-		tr.LogFields(
-			otlog.Object("args", args),
-			otlog.String("ensure_revision", req.EnsureRevision),
+		tr.AddEvent(
+			"LogFields",
+			attribute.String("args", args),
+			attribute.String("ensure_revision", req.EnsureRevision),
```

Comment:
Looks like these might be attributes of a trace, not an event

```suggestion
		tr.SetAttributes(
			attribute.String("args", args),
			attribute.String("ensure_revision", req.EnsureRevision),
```